src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/seo.html.twig

```diff
@@ -0,0 +1,60 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+<div role="tabpanel" class="form-contenttab tab-pane container-fluid" id="seo-tab">
+  <h2>{{ 'Search Engine Optimization'|trans({}, 'Admin.Catalog.Feature') }}</h2>
+  <p class="subtitle">{{ 'Improve your ranking and how your product page will appear in search engines results.'|trans({}, 'Admin.Catalog.Feature') }}</p>
+
+  {% block product_catalog_tool_serp %}
+    <p>{{ "Here is a preview of your search engine result, play with it!"|trans({}, 'Admin.Catalog.Feature') }}</p>
+    {# Div targetted by the SERP component in VueJs. It displays a Google search result preview. #}
+    <div id="serp-app"></div>
+  {% endblock %}
+
+  {{ form_widget(productForm.seo) }}
+
+  <div class="row">
+    <div class="col-md-9">
+      <div class="alert alert-info" role="alert">
+        <p class="alert-text">
+          {% if configuration('PS_REWRITING_SETTINGS') == 0 %}
```

Comment:
Idea: introduce twig functions `configurationIsDisabled()` and `configurationIsEnabled()` to remove the `== 0` and `== 1`
```suggestion
          {% if configurationIsDisabled('PS_REWRITING_SETTINGS') %}
```
Might be over-engineering 🤔  ? Let me know

Reply:
I just copied this so out of scope as well, but it might a good idea Although I would make it simpler to avoid two functions, just one which returns a boolean should be enough  `if configurationIsEnabled('PS_REWRITING_SETTINGS')` `if not configurationIsEnabled('PS_REWRITING_SETTINGS')`

Reply:
can't we just make sure that it's boolean and do something like:

`!...` or `var not eq true`?

Reply:
I thought the `!` didn't work in twig 🤔

Reply:
@jolelievre you're right, in that case I don't see anything wrong with `== 0` :D